db/schema.rb

```diff
@@ -59,18 +59,19 @@
     t.date     "birthday"
     t.string   "phone"
     t.string   "password_digest"
-    t.datetime "created_at",                      null: false
-    t.datetime "updated_at",                      null: false
+    t.datetime "created_at",                           null: false
+    t.datetime "updated_at",                           null: false
     t.string   "user_type"
     t.string   "provider"
     t.string   "oauth_id"
     t.string   "confirm_token"
-    t.boolean  "confirmed",       default: false
+    t.boolean  "confirmed",            default: false
     t.string   "state"
     t.string   "city"
     t.string   "street_address"
     t.string   "image_url"
-    t.boolean  "has_taken_quiz",  default: false
+    t.boolean  "has_taken_quiz",       default: false
+    t.boolean  "enable_notifications", default: false
```

Comment:
The enable_notifications column should be default to true, that way notifications are only sent to the user unless he decides to disable them